File: Homeworks_Solutions/12_Java_Basic_Web_Exercises/12_Java_Basic_Web_Calculator-ver.2/src/main/java/com/softuni/entity/Calculator.java
package com.softuni.entity;

public class Calculator {
    private double leftOperand;
    private double rightOperand;
    private String operator;

    public Calculator(double leftOperand, double rightOperand, String operator) {
        this.leftOperand = leftOperand;
        this.rightOperand = rightOperand;
        this.operator = operator;
    }

    public double getLeftOperand() {
        return leftOperand;
    }

    public void setLeftOperand(double leftOperand) {
        this.leftOperand = leftOperand;
    }

    public double getRightOperand() {
        return rightOperand;
    }

    public void setRightOperand(double rightOperand) {
        this.rightOperand = rightOperand;
    }

    public String getOperator() {
        return operator;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public double calculateResult() {

        double result;
		String errorMsg = "";
        switch (this.operator){

            case "+":
                result = this.leftOperand + this.rightOperand;
                break;
				
            case "-":
                result = this.leftOperand - this.rightOperand;
                break;
				
            case "*":
                result = this.leftOperand * this.rightOperand;
                break;
				
            case "/":
                result = this.leftOperand / this.rightOperand;
                break;
				
			case "^":
                result = Math.pow(this.leftOperand, this.rightOperand);
                break;
				
			case "%":
			    result = this.leftOperand % this.rightOperand;
                break;

            case "OR":
                result = (int)(this.leftOperand) | (int)(this.rightOperand);
                break;

            case "AND":
                result = (int)(this.leftOperand) & (int)(this.rightOperand);
                break;

            case "XOR":
                result = (int)(this.leftOperand) ^ (int)(this.rightOperand);
                break;

            default:
                result = 0;
                break;
        }
        return result;
    }
}

File: Homeworks_Solutions/09_PHP_MVC_Symfony_Overview_Exercises/Calculator_Update/src/CalculatorBundle/Entity/Calculator.php
<?php
namespace CalculatorBundle\Entity;

class Calculator
{
    // TODO add class fields and properties, getters and setters
    
    /**
     * @var float
     */
    private $leftOperand;

    /**
     * Get left operand
     * 
     * @return float
     */
    public function getLeftOperand(){
        return $this->leftOperand;
    }

    /**
     * Set left Operand
     * 
     * @param float $leftOperand
     * 
     * @return Calculator
     */
    public function setLeftOperand ($operand)
    {
        $this->leftOperand = $operand;

        return $this;
    }

    /**
     * @var float
     */
    private $rightOperand;

    /**
     * Get right operand
     * 
     * @return float
     */
    public function getRightOperand(){
        return $this->rightOperand;
    }
    /**
     * Set left Operand
     * 
     * @param float $rightOperand
     * 
     * @return Calculator
     */
    public function setRightOperand ($operand)
    {
        $this->rightOperand = $operand;
        
        return $this;
    }
    /**
     * @var string
     */
    private $operator;

    /**
     * Get operator
     * 
     * @return string
     */
    public function getOperator(){
        return $this->operator;
    }
    /**
     * Set operator
     * 
     * @param string $operator
     * 
     * @return Calculator
     */
    public function setOperator ($operator)
    {
        $this->operator = $operator;
        
        return $this;
    }

}